CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+### v2.0.0-beta1 -> v2.0.0-beta2
+
+#### Features
+
+ - [`a713e13`](https://github.com/deis/slugbuilder/commit/a713e139983995b9f8bdd424497f5fd7683aca03) storage: use the objectstorage cli for upload and download of slug
+ - [`71a4ed2`](https://github.com/deis/slugbuilder/commit/71a4ed22b32d2d74e7ef25d3c82eb3b21e283236) _scripts: add CHANGELOG.md and generator script
+ - [`9ab1265`](https://github.com/deis/slugbuilder/commit/9ab126507c86a476633f84cb0a23d67c20e1f9c8) storage: Add support for multiple object storages
+
+#### Fixes
+
+ - [`c55ef21`](https://github.com/deis/slugbuilder/commit/c55ef213fd4538cfe1bc4b7584e4d19af55a39d4) objectstorecli: the latest version having the s3 auth fix
+
+#### Documentation
```

Comment:
Can you remove this empty "Documentation" section? The script isn't smart enough not to include it. :-/